rst_files/version_control.rst

```diff
@@ -39,7 +40,7 @@ Setup
 
     2. Install the `GitHub Desktop <https://desktop.github.com/>`_ application
 
-3. Optionally (but strongly recommended):  On Windows, change the default line-ending by
+3. Optionally (but strongly recommended):  On Windows, change the default line-ending by:
```

Comment:
I thought the style was no-colons/minimal punctuation. Don't mind either way, though.

Reply:
True - but in this case, I think I found it confusing because there's it's a list in a list... not sure

Reply:
Yeah, I have no issue with it. Guess we can keep it for now.